packages/modelserver-node/src/client/model-server-client.ts

```diff
@@ -416,6 +416,10 @@ class DefaultTransactionContext implements TransactionContext {
             return Promise.reject('Socket is closed.');
         }
 
+        if (!patch || (Array.isArray(patch) && patch.length === 0)) {
+            return { success: false };
```

Comment:
why is it no success applying an empty patch?

Reply:
That's meant to signal to the caller that it was a nonsensical request. The server didn't attempt to apply the patch, so the patch was not successful.